### ISSUE TITLE
Better name for "Note or Remark"

### ISSUE THREAD

**Issue by sz332** [gaphor/gaphor#2589]
### Is your feature request related to a problem?

Every time I see it I ask the question: is it note or remark? What should go here? 

### Describe the solution you'd like

Let's find a better name for this. Other tools I have seen use simply the term "Documentation" but open for suggestions.

**Comment by mikekidner** [comment]
"Documentation" or "Description"provides clearer guidance to the user to add some useful information about the element.
Pulling this text from the model can then be used for system description documents, using `element.note`
Which goes to issue #2582 re text formatting.

**Comment by sz332** [comment]
@amolenaar @danyeaw  what do you think about Documentation (or Description)?

**Comment by danyeaw** [comment]
Documentation I guess would be OK with me, I feel like the whole element is the "documentation". Maybe Extra Info?

**Comment by sz332** [comment]
Yeah, I was thinking about the same related to "documentation", but what about Description?

**Comment by danyeaw** [comment]
The original words, note or remark mean to me any type of info you want to capture. Description is less general, that we only want users to use that field to describe the element. Maybe that is OK and a best practice we want to use the field for, but it may confuse users who want to capture something else.

**Comment by sz332** [comment]
Can you provide an example what else than an element description the users might want to enter there? I mean, this information is actually **part** of the model element.

**Comment by mikekidner** [comment]
If it is a free-form field there are many different ways it could get used, for instance:

- A URL for the datasheet, 
- A link to code repo
- A reviewers comment/approval
- A full text description of the element
- A list of changes
- Message to another team member

It may not be the intended use, and each group of users will have their own workflows and conventions. 

It could be that "Note" or "Free text" is sufficient, and some context help to say it is free-form text and will stick with the element in the model.

**Comment by sz332** [comment]
What about Note or Notes then?

**Comment by mikekidner** [comment]
"Note" makes sense (and the underlying attribute is ".note"). 
I've made this change in a pull request. However I think that branch had some unintended changes from attempts to fix floating execution nodes, which I thought I'd got rid off. Sorry.

**Comment by danyeaw** [comment]
Yup, I agree that Note is a good one.

**Comment by sz332** [comment]
Let's go with Note then.

**Comment by amolenaar** [comment]
Fixed by #2596.
